src/core/AnimatedEvent.js

```diff
@@ -27,23 +74,59 @@ function sanitizeArgMapping(argMapping) {
   );
 
   // Assume that the event containing `nativeEvent` is always the first argument.
-  traverse(argMapping[0].nativeEvent, []);
+  const ev = argMapping[0].nativeEvent;
+  if (typeof ev === 'object') {
+    traverse(ev, []);
+  } else if (typeof ev === 'function') {
+    const proxyHandler = {
+      get: function(target, name) {
+        if (name === '__isProxy') {
+          return true;
+        }
+        if (!target[name] && name !== '__val') {
+          target[name] = new Proxy({}, proxyHandler);
+        }
+        return target[name];
+      },
+      set: function(target, prop, value) {
+        if (prop === '__val') {
+          target[prop] = value;
+        }
+      },
+    };
 
-  return eventMappings;
+    const proxy =
+      Platform.OS === 'android'
+        ? androidProxyPolyfill()
+        : new Proxy({}, proxyHandler);
+    alwaysNodes.push(new AnimatedAlways(ev(proxy)));
```

Comment:
Is if ok to use always node here? Can you verify that if we use it for a scrollview it won't be running updates when scroll view events does not get triggered? (e.g. place a `debug` node inside event handler code and make sure logs are not showing up until we scroll)

Reply:
Yes, it's ok